cli/utils.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+func parseFile(filePath string, v protoreflect.ProtoMessage) error {
+	b, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return err
+	}
+
+	switch filepath.Ext(filePath) {
+	case ".json":
+		if err := protojson.Unmarshal(b, v); err != nil {
+			return fmt.Errorf("invalid json: %w", err)
+		}
+	case ".yaml", ".yml":
+		if err := protojson.Unmarshal(b, v); err != nil {
+			return fmt.Errorf("invalid yaml: %w", err)
+		}
+	default:
+		return errors.New("unsupported file type") //nolint
+	}
+
+	return nil
+}
+
+func prettyPrint(i interface{}) string {
+	s, e := json.MarshalIndent(i, "", "\t")
```

Comment:
Use `protojson` for correct representation of enums as string.

Reply:
Couldn't find an indentation function in `protojson`. It has `MarshOptions` with indentation, but it supports `protoreflect.ProtoMessage` inputs only.

```	
      marshalOpts:= protojson.MarshalOptions{
		Indent: "\t",
	}
```

`func (protojson.MarshalOptions).Marshal(m protoreflect.ProtoMessage) ([]byte, error)`

Reply:
All message types in go generated protobuf code satisfy the `protoreflect.ProtoMessage` interface.
So we should be able to use this.